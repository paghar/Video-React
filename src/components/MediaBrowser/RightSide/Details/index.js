//@flow
import React from 'react';
import Typography from 'components/shared/Typography';
import { Display4 } from 'theme/fonts';
import { Generals } from 'theme/colors';
import { DetailWrapper } from './index.style';
const Details = () => {
  return (
    <DetailWrapper>
      <Typography font={Display4} hoverable color={Generals.vi}>
        Details
      </Typography>
    </DetailWrapper>
  );
};

export default Details;
