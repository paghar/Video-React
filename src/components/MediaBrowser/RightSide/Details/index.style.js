//@flow
import styled from 'styled-components';
import { Generals } from 'theme/colors';

export const DetailWrapper = styled.div`
  height: 100%;
  overflow-y: auto;
  overflow-x: hidden;
  ::-webkit-scrollbar {
    width: 2px;
    background-color: ${Generals.v};
  }
  ::-webkit-scrollbar-thumb {
    border-radius: 10px;
    background-color: ${Generals.vi};
    width: 3px;
    opacity: 0.3;
  }
  ::-webkit-scrollbar-track {
    -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
    background-color: ${Generals.v};
    margin-top: 5px;
    border-radius: 10px;
    opacity: 0.3;
    padding: 5px 0px;
  }
`;
