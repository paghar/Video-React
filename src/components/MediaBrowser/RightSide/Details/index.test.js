//@flow
import React from 'react';
import Details from '.';
import renderer from 'react-test-renderer';
import { create } from 'react-test-renderer';
describe('<Details />', () => {
  it('checks the UI', () => {
    const wrapper = renderer.create(<Details />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
