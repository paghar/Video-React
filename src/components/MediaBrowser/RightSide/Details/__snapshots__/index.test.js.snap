// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Details /> checks the UI 1`] = `
.c1 {
  pointer-events: none;
  -moz-user-select: none;
  -khtml-user-select: none;
  -webkit-user-select: none;
  -ms-user-select: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.c0 {
  height: 100%;
  overflow-y: auto;
  overflow-x: hidden;
}

.c0::-webkit-scrollbar {
  width: 2px;
}

.c0::-webkit-scrollbar-thumb {
  border-radius: 10px;
  width: 3px;
  opacity: 0.3;
}

.c0::-webkit-scrollbar-track {
  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
  margin-top: 5px;
  border-radius: 10px;
  opacity: 0.3;
  padding: 5px 0px;
}

<div
  className="c0"
>
  <span
    className="c1"
  >
    Details
  </span>
</div>
`;
